cypress-smoketests/integration/trial-sessions/trial-session-working-copy.spec.js

```diff
@@ -151,24 +159,25 @@ describe('Petitions Clerk', () => {
   });
 });
 
-describe('Judge', () => {
+describe.skip('Judge', () => {
```

Comment:
Why are we skipping this?

Reply:
Story #4830 was in progress. It'll be re-enabled in the next PR.